python/lookout/sdk/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.2.0"
```

Comment:
It is more common to specify this as a tuple:
```python
__version__ = 0, 2, 0
```

Reply:
I also thought that but apparently, it's not true.

Here is example of 4 different VERY common libs all of them use string version:
https://github.com/urllib3/urllib3/blob/1.24.1/src/urllib3/__init__.py#L30
https://github.com/requests/requests/blob/master/requests/__version__.py#L8
https://github.com/chardet/chardet/blob/master/chardet/version.py#L8
https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/__init__.py#L133

Though we can add `VERSION` variable also as [chardet](https://github.com/chardet/chardet/blob/master/chardet/version.py#L9)

Reply:
LOL you are right https://www.python.org/dev/peps/pep-0396
A good example of poor design... I have to live with it

Reply:
For reference, another example of string version can be found in the docs here: https://packaging.python.org/tutorials/packaging-projects/#creating-setup-py

Reply:
Wow! The PEP is from 2011! But I have never heard of it!